### PR TITLE
Pipeline fixes

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/workflows/publish_docker_images_cron.yml [hunk 1]
```diff
@@ -20,15 +20,18 @@ jobs:
...
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
+      - name: DockerHub Login
+        run: docker login -u ${{ secrets.DOCKERHUB_USER_NAME }} -p ${{ secrets.DOCKERHUB_PASSWORD }}
+
       - name: Build and publish docker images
         run: |
           python -m  tools.packaging_automation.publish_docker  --pipeline_trigger_type "${GITHUB_EVENT_NAME}"\
-           --exec_path "$(pwd)" --image_type ${{ matrix.image_type }}
+          --exec_path "$(pwd)" --image_type ${{ matrix.image_type }}
```
Comment (onurctirtir) [inline .github/workflows/publish_docker_images_cron.yml:37]: I think we should remove `--exec-path`   param from all the scripts and prefer the github actions based approach that you shared in other pr, but this is not related with this pr.
Can you open an issue for that in tools repo ?

File: .github/workflows/publish_docker_images_cron.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: Schedule Build Docker Images For Main Images And Nightly
 
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: '*/10 * * * *'
```
Comment (onurctirtir) [inline .github/workflows/publish_docker_images_cron.yml:5]: that would run at every 10th minute, is that really what we want ?

https://crontab.guru/#*/10_*_*_*_*

File: .github/workflows/publish_docker_images_cron.yml (hunk 1 above)
Comment (onurctirtir) [reply .github/workflows/publish_docker_images_cron.yml:37]: Discussed in our private chat and seems that I misunderstood the usage of this param.

However, we still need to introduce this to README.

File: .github/workflows/publish_docker_images_cron.yml (hunk 2 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_cron.yml]: Fixed and changed to execute at 00:30 everyday

File: .github/workflows/publish_docker_images_cron.yml (hunk 1 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_cron.yml]: Added in readme in tools repo

File: .github/workflows/publish_docker_images_cron.yml [hunk 3]
```diff
@@ -20,15 +20,20 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Clone tools branch
-        run: git clone --depth=1 --branch update_properties_fixes https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_cron.yml:23]: I suggest you create a tag on tools repo, and use that instead.

Using develop branch is prone to errors in the future. Also tags are cheap, let's bump a new version on tools.

File: .github/workflows/publish_docker_images_on_push.yml [hunk 4]
```diff
@@ -20,15 +20,18 @@ jobs:
...
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
+      - name: DockerHub Login
+        run: docker login -u ${{ secrets.DOCKERHUB_USER_NAME }} -p ${{ secrets.DOCKERHUB_PASSWORD }}
+
       - name: Build and publish docker images
         run: |
               python -m  tools.packaging_automation.publish_docker --github_ref "${GITHUB_REF}" \
-              --pipeline_trigger_type "${GITHUB_EVENT_NAME}" --exec_path "$(pwd)" --image_type ${{ matrix.image_type }}
+              --pipeline_trigger_type "${GITHUB_EVENT_NAME}"  --image_type ${{ matrix.image_type }}
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_on_push.yml:37]: please have one arg per line

File: .github/workflows/publish_docker_images_on_manual.yml [hunk 5]
```diff
@@ -20,16 +20,19 @@ jobs:
...
 
+      - name: DockerHub Login
+        run: docker login -u ${{ secrets.DOCKERHUB_USER_NAME }} -p ${{ secrets.DOCKERHUB_PASSWORD }}
+
       - name: Build and publish docker images
         run: |
             python -m  tools.packaging_automation.publish_docker  --pipeline_trigger_type "${GITHUB_EVENT_NAME}" \
-            --exec_path "$(pwd)" --tag_name ${{ github.event.inputs.tag_name }} \
+             --tag_name ${{ github.event.inputs.tag_name }} \
             --manual_trigger_type ${{ github.event.inputs.trigger_type }}
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_on_manual.yml:38]: inconsistent spacing.
please have one arg per line

**Comment by hanefi** [comment]
I'd expect the pipeline to get triggered on this PR. Why is this not the case here?

File: .github/workflows/publish_docker_images_on_push.yml (hunk 4 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_on_push.yml:37]: Fixed

File: .github/workflows/publish_docker_images_on_manual.yml (hunk 5 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_on_manual.yml:38]: Fixed

**Comment by gurkanindibay** [comment]
> I'd expect the pipeline to get triggered on this PR. Why is this not the case here?

Because all of the pipelines are disabled now. I can enable them for you to see the execution

File: .github/workflows/publish_docker_images_cron.yml [hunk 6]
```diff
@@ -2,7 +2,7 @@ name: Schedule Build Docker Images For Main Images And Nightly
 
 on:
   schedule:
-    - cron: '0 0 * * *'
+    - cron: 30 0 * * *
```
Comment (hanefi) [inline .github/workflows/publish_docker_images_cron.yml:5]: I understand how we are conditioned to schedule cron jobs around midnight, even though this setting is in UTC time.

( FYI, I am writing this in 00:30 UTC = 3:30 TRT )

**Review by hanefi — APPROVED** [review]

File: .github/workflows/publish_docker_images_cron.yml (hunk 6 above)
Comment (gurkanindibay) [reply .github/workflows/publish_docker_images_cron.yml:5]: By the way cron schedules trigger only main branch. They are not triggered on non-main branches https://stackoverflow.com/questions/63436541/github-action-workflow-schedule-not-working-on-non-default-branch
